capstone/src/main/webapp/script.js

```diff
@@ -316,20 +306,28 @@ async function averagePrecinctSentiment(policePrecinct) {
     }
   }
   const sentimentAverage = sentimentCount/list.length;
-  return getSentimentColor(sentimentAverage);
+  return getSentimentColor(sentimentAverage, policePrecinct);
 }
 
-function getSentimentColor(averageFeelings) {
+function getSentimentColor(averageFeelings, policePrecinct) {
+  let precinctColor = '#228B22';
   if (Math.round(averageFeelings) == 5) {
```

Comment:
Could you add a comment or explanation why you don't need to consider averageFeelings above 5?

Reply:
I added documentation for that function. Please let me know if the explanation is not clear.